src/exports.cc

```diff
@@ -925,5 +932,31 @@ Word get_log_level(Word result_level_uint32_ptr) {
   return WasmResult::Ok;
 }
 
+Word wasi_unstable_fd_fdstat_set_flags(Word /*fd*/, Word /*flags*/) {
```

Comment:
1. Suggest situating these alongside the other wasi_unstable_* hostcall definitions.
2. The other hostcall definitions include a comment that spells out the specific types (since `Word` can represent various different types of values), e.g. https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/63cb9c1a40d8e6382d3f302721e09b8c45466661/src/exports.cc#L865-L866. For consistency and completeness, suggest doing the same for these new hostcalls.

Reply:
Done.